doc/src/base/base.md

```diff
@@ -37,6 +37,22 @@ ans
 
 ## Keywords
 
+This is the list of reserved keywords in Julia:
+`baremodule`, `begin`, `break`, `catch`, `const`, `continue`, `do`,
+`else`, `elseif`, `end`, `export`, `false`, `finally`, `for`, `function`,
+`global`, `if`, `import`, `let`, `local`, `macro`, `module`, `quote`,
+`return`, `struct`, `true`, `try`, `using`, `while`.
+Those keywords are not allowed to be used as variable names.
+
+The following two word sequences are reserved:
+`abstract type`, `mutable struct`, `primitive type`.
```

Comment:
three word sequences

Reply:
It is better to just open a new PR with the suggested changes instead of commenting on old merged PRs.

Reply:
Thank you. I will make a PR to clarify it.